File: leetcode/string/easy/reverse-string.cpp
class Solution {
public:
    void reverseString(vector<char>& s) {
        int n = s.size();
        int m = n/2;
        for(int i=0;i<m;i++){
            int temp = s[i];
            s[i] = s[n - i - 1];
            s[n - i - 1] = temp;
        }
    }
};

File: leetcode/string/easy/reverse-string.py
class Solution(object):
    def reverseString(self, s):
        """
        :type s: List[str]
        :rtype: None Do not return anything, modify s in-place instead.
        """
        right = 0
        left = len(s)-1
        while right < left:
            s[right], s[left] = s[left], s[right]
            left -= 1
            right += 1